src/Altinn.Profile/Models/GroupModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Altinn.Profile.Models
+{
+    /// <summary>
+    /// GroupModel is used to represent a group of parties
+    /// </summary>
+    public class GroupModel
```

Comment:
The only use for this class is as a base class for GroupResponse. I assume it's going to be used by other call chains?